internal/interpreter/Cargo.toml

```diff
@@ -44,18 +44,32 @@ std = ["i-slint-core/std"]
 ## backend will not be operational
 backend-qt = ["i-slint-backend-selector/i-slint-backend-qt", "i-slint-backend-selector/rtti-qt", "std"]
 
-## The GL backend uses the `winit` crate for the windowing system integration,
-## and the `femtovg` crate for the rendering. With this feature, all windowing
-## systems are supported. For a smaller build, omit this feature and select
-## one of the other specific `backend-gl-XX` features.
-backend-gl-all = ["i-slint-backend-selector/backend-gl-all", "i-slint-backend-selector/rtti-gl", "std"]
-## Simliar to `backend-gl-all` this enables the GL backend but only with support for the
-## X Window System on Unix.
-backend-gl-x11 = ["i-slint-backend-selector/backend-gl-x11", "i-slint-backend-selector/rtti-gl", "std"]
-## Simliar to `backend-gl-all` this enables the GL backend but only with support for the
-## Wayland window system on Unix.
-backend-gl-wayland = ["i-slint-backend-selector/backend-gl-wayland", "i-slint-backend-selector/rtti-gl", "std"]
+## This feature is an alias for the `eventloop-winit` feature and `renderer-femtovg`
+backend-gl-all = ["eventloop-winit", "renderer-femtovg"]
+backend-gl-x11 = ["eventloop-winit-x11", "renderer-femtovg"]
+backend-gl-wayland = ["eventloop-winit-wayland", "renderer-femtovg"]
```

Comment:
Each of them need a documentation or they won't be shown in the doc.

In fact, since it is just for compatibility, none of them really needs doc, then you can use a simple '#'